gitlab/gitlab-eslint.yaml

```diff
@@ -0,0 +1,151 @@
+rules:
+- id: eslint.detect-disable-mustache-escape
+  languages:
+  - javascript
+  - typescript
+  message: |
+    Markup escaping disabled. This can be used with some template engines to escape
+    disabling of HTML entities, which can lead to XSS attacks.
+  metadata:
+    category: security
+    cwe: 'CWE-116: Improper Encoding or Escaping of Output'
+    owasp: 'A7: Cross-Site Scripting XSS'
+    source-rule-url: https://github.com/nodesecurity/eslint-plugin-security/blob/master/rules/detect-disable-mustache-escape.js
+    technology:
+    - wkhtmltopdf
```

Comment:
Similar issue here, most of these list `wkhtmltopdf`. This isn't a big deal for this particular PR, but looks like something we should fix.